### PR TITLE
[Stmt] Add rowCount column to SHOW DATA stmt

### LINKED ISSUE #3675: [Stmt][Show Data] show the row count of specified materialized index
**Is your feature request related to a problem? Please describe.**

In the current query planning, we will use the number of rows as the basis for selecting the materialized view. But there is currently no statement that can show the number of statistical rows of each materialized view

**Describe the solution you'd like**
Add row count in `SHOW DATA` stmt's result.

### PR DESCRIPTION
**Pull request by morningman**
User can see the row count of all materialized indexes of a table.

```
mysql> show data from test;
+-----------+-----------+-----------+--------------+----------+
| TableName | IndexName | Size      | ReplicaCount | RowCount |
+-----------+-----------+-----------+--------------+----------+
| test2     | r1        | 10.000MB  | 30           | 10000    |
|           | r2        | 20.000MB  | 30           | 20000    |
|           | test2     | 50.000MB  | 30           | 50000    |
|           | Total     | 80.000    | 90           |          |
+-----------+-----------+-----------+--------------+----------+
```

Fix #3675

### CODE REVIEW

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md [hunk 1]
```diff
@@ -25,23 +25,65 @@ under the License.
...
-    该语句用于展示数据量和副本数量
-    语法：
-        SHOW DATA [FROM db_name[.table_name]];
-        
-    说明：
-        1. 如果不指定 FROM 子句，使用展示当前 db 下细分到各个 table 的数据量和副本数量
-        2. 如果指定 FROM 子句，则展示 table 下细分到各个 index 的数据量和副本数量
-        3. 如果想查看各个 Partition 的大小，请参阅 help show partitions
+
+该语句用于展示数据量、副本数量已经统计行数。
```
Comment (kangkaisen) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:31]:
```suggestion
该语句用于展示数据量、副本数量以及统计行数。
```

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md [hunk 2]
```diff
@@ -25,23 +25,65 @@ under the License.
...
+    +-----------+-------------+--------------+
+    | tbl1      | 900.000 B   | 6            |
+    | tbl2      | 500.000 B   | 3            |
+    | Total     | 1.400 KB    | 9            |
+    | Quota     | 1024.000 GB | 1073741824   |
+    | Left      | 1021.921 GB | 1073741815   |
+    +-----------+-------------+--------------+
+    ```
+    
+2. 展示指定 db 的下指定表的细分数据量和副本数量
```
Comment (kangkaisen) [inline docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:71]:
```suggestion
2. 展示指定 db 的下指定表的细分数据量和副本数量和统计行数
```

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md (hunk 1 above)
Comment (morningman) [reply docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:31]: Done

File: docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md (hunk 2 above)
Comment (morningman) [reply docs/zh-CN/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:71]: Done

**Review by kangkaisen — APPROVED** [review]
+1, LGTM

File: docs/en/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md [hunk 3]
```diff
@@ -25,22 +25,62 @@ under the License.
 -->
 
 # SHOW DATA
+
 ## Description
-This statement is used to show the amount of data and the number of replica
-Grammar:
+
+This statement is used to show the amount of data, the number of replica and num of rows.
```
Comment (wutiangan) [inline docs/en/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:31]:
```suggestion
This statement is used to show the amount of data, the number of replica and the number of rows.
```

Comment (morningman) [reply docs/en/sql-reference/sql-statements/Data Manipulation/SHOW DATA.md:31]: I will modify this is another PR

**Pull request merged**